caravel/models.py

```diff
@@ -412,6 +412,8 @@ class Database(Model, AuditMixinNullable):
     cache_timeout = Column(Integer)
     select_as_create_table_as = Column(Boolean, default=False)
     expose_in_sqllab = Column(Boolean, default=False)
+    allow_run_sync = Column(Boolean, default=True)
```

Comment:
i read a little further and it looks like `sync` is the default, in which case we could store just one column `allow_run_async` and if that is true run async, if false run sync. seems simpler to me.